tests-utils/src/main/scala/tech.beshu.ror.utils/containers/ReadonlyRestEsClusterContainer.scala

```diff
@@ -42,15 +42,17 @@ object ReadonlyRestEsCluster {
                                   nodeDataInitializer: ElasticsearchNodeDataInitializer = NoOpElasticsearchNodeDataInitializer,
                                   clusterInitializer: ReadonlyRestEsClusterInitializer = NoOpReadonlyRestEsClusterInitializer,
                                   dependentServicesContainers: List[DependencyDef] = Nil,
-                                  configHotReloadingEnabled: Boolean = true): ReadonlyRestEsClusterContainer =
+                                  configHotReloadingEnabled: Boolean = true,
+                                  internodeSslEnabled: Boolean = false): ReadonlyRestEsClusterContainer =
```

Comment:
maybe it's the time, when we should introduce some config class to gather all additional settings of ROR container?